modules/core/04-channel/keeper/grpc_query_test.go

```diff
@@ -956,6 +985,22 @@ func (suite *KeeperTestSuite) TestQueryPacketAcknowledgement() {
 			},
 			false,
 		},
+		{
+			"ack not found",
+			func() {
+				path := ibctesting.NewPath(suite.chainA, suite.chainB)
+				path.Setup()
+				expAck = []byte("hash")
+				suite.chainA.App.GetIBCKeeper().ChannelKeeper.SetPacketAcknowledgement(suite.chainA.GetContext(), path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID, 1, expAck)
+
+				req = &types.QueryPacketAcknowledgementRequest{
+					PortId:    "test-port-id",
+					ChannelId: "test-channel-id",
```

Comment:
We should use valid IDs here , right? Otherwise the function would error in the `HasChannel` check, right?

```suggestion
					PortId:    path.EndpointA.ChannelConfig.PortID,
					ChannelId: path.EndpointA.ChannelID,
```